django/db/backends/postgresql/base.py

```diff
@@ -379,11 +413,43 @@ def make_debug_cursor(self, cursor):
         return CursorDebugWrapper(cursor, self)
 
 
-class CursorDebugWrapper(BaseCursorDebugWrapper):
-    def copy_expert(self, sql, file, *args):
-        with self.debug_sql(sql):
-            return self.cursor.copy_expert(sql, file, *args)
+if is_psycopg3:
+
+    class Cursor(Database.Cursor):
+        """
+        A subclass of psycopg cursor implementing callproc.
+        """
+
+        def callproc(self, name, args=None):
+            if not isinstance(name, sql.Identifier):
+                name = sql.Identifier(name)
+
+            qparts = [sql.SQL("select * from "), name, sql.SQL("(")]
```

Comment:
```suggestion
            qparts = [sql.SQL("SELECT * FROM "), name, sql.SQL("(")]
```
I aligned the case with the rest of the SQL code.